src/app/build/build.component.html

```diff
@@ -77,6 +77,10 @@ <h1>
             <strong>{{ jobs.length }}</strong> jobs ran.
           </span>
         </span>
+
+        <pre>
+{{ buildlogs }}
```

Comment:
One thing here - can we have any sort of text wrapping for the build log?

<img width="953" alt="buildlogs" src="https://user-images.githubusercontent.com/13103165/42822917-e9f8c2a2-89e4-11e8-98a6-25e21c8eb1df.PNG">

Reply:
Feedback incorporated @radu-matei - thanks so much. Build logs are visible now, and i've added text wrapping styles.